readme.md

```diff
@@ -16,114 +15,98 @@ Member States (from the European Economic Area, which encompasses more countries
-
-
-## HCERT
-
-[Explain that HCERT implementation contains the entire stack]
-
-[TODO half of these repos contain an almost full stack impelementain]
-
-https://github.com/ehn-dcc-development/hcert-kotlin
-https://github.com/ehn-dcc-development/hcert-app-swift
-https://github.com/ehn-dcc-development/python-hcert
-https://github.com/ehn-dcc-development/hcert-service-kotlin
-https://github.com/ehn-dcc-development/hcert-dotnet
-https://github.com/ehn-dcc-development/hcert-java
-
-## COSE
-
-[Explain]
-
-https://github.com/ehn-dcc-development/cose-js
-https://github.com/ehn-dcc-development/COSE-JAVA
-
-
-## Base45
+## Specifications
 
-[Explain]
+The DCC specifications have been split into four repositiros:
 
-https://github.com/ehn-dcc-development/base45-php
-https://github.com/ehn-dcc-development/base45-cs
-https://github.com/ehn-dcc-development/base45-ansi-C
-https://github.com/ehn-dcc-development/base45-swift
-https://github.com/ehn-dcc-development/base45-js
-https://github.com/ehn-dcc-development/base45-java
+* [hcert-spec](https://github.com/ehn-dcc-development/hcert-spec) specifies the container and encoding formats used by the DCC.
+* [ehn-dcc-valuesets](https://github.com/ehn-dcc-development/ehn-dcc-valuesets) provides the valueset datastructures and a snapshot of the valuesets (the current versions are published on the DCCG Gateway.
+* [ehn-dcc-schema](https://github.com/ehn-dcc-development/ehn-dcc-schema) specifies the DCC schema (the specification of the data stored within the DCC).
+* [dgc-business-rules](https://github.com/ehn-dcc-development/dgc-business-rules) specifies the Business Rules framework.
 
+You can find these repositories pinned on our github [homepage](https://github.com/ehn-dcc-development).
 
-## Verifiers
+We have a team of maintainers who monitor the above mentioned pinned repositories. If you have questions or suggestions please feel free to open an issue on the respective repository.
 
-[Explain]
+## Maintainers
 
-https://github.com/ehn-dcc-development/DGCValidator | Xamarin scanner app (Android/iOS)
+The core repoisotires are maintained a small group of people attached to EU Member States or the European Commission. 
 
-https://github.com/ehn-dcc-development/ValidationCore | Swift/Android scanner libraries
+Repository          | Teams
+------------------- | ------------
+hcert-spec          | [hcert-spec-editors](https://github.com/orgs/ehn-dcc-development/teams/hcert-spec-editors)
+ehn-dcc-valuesets   | [ehn-valueset-editors](https://github.com/orgs/ehn-dcc-development/teams/ehn-valueset-editors)
+ehn-dcc-schema      | [ehn-schema-editors](https://github.com/orgs/ehn-dcc-development/teams/ehn-schema-editors)
+dgc-business-rules  | [ehn-business-rules-editors](https://github.com/orgs/ehn-dcc-development/teams/ehn-business-rules-editors)
+overview            | [overview-editors](https://github.com/orgs/ehn-dcc-development/teams/overview-editors)
```

Comment:
Maybe put `overview` upfront?

Reply:
Do you want to add repos like `dcc-business-rules-analysis`, etc. here as well?

Reply:
Yeah, if we finish this PR, then you can add anything I've missed? :)